monai/handlers/tensorboard_handlers.py

```diff
@@ -173,6 +173,21 @@ def iteration_completed(self, engine: Engine) -> None:
         else:
             self._default_iteration_writer(engine, self._writer)
 
+    def _write_scalar(self, engine: Engine, writer: SummaryWriter, tag: str, value: Any, step: int) -> None:
+        """
+        Write scale values into TensorBoard.
```

Comment:
Should "scale values" be "a scalar"?

Reply:
Sorry for the typo, updated in the latest commit.

Thanks.